templates/base.html

```diff
@@ -47,102 +48,114 @@
         {% block extrastyle %}{% endblock %}
     </head>
     <body>
-        <div class="off-canvas-wrapper">
-            <div class="off-canvas-wrapper-inner" data-off-canvas-wrapper>
-                <div class="off-canvas position-right" id="offCanvas" data-off-canvas data-position="right">
-                    <!-- Close button -->
-                    <button class="close-button" aria-label="Close menu" type="button" data-close>
-                        <span aria-hidden="true">&times;</span>
-                    </button>
+        {% if user.is_authenticated %}
```

Comment:
Going to re-think how this part will work. 

The goal is to re-direct people to their profile instead of the about page once they have logged in. This way un-auth users see the "about" and then "auth" users just see their profile as the home. 

I think that once someone is logged in, it makes sense to cater the environment more toward their profile and discovering instead of learning about perdiem.